app/src/main/res/values-fr/strings.xml

```diff
@@ -74,7 +74,7 @@
         <item quantity="many">%1$d / %2$d Fichiers</item>
         <item quantity="other">%1$d / %2$d Fichiers</item>
     </plurals>
-    <string name="override_changes">Ignorer les modifications</string>
+    <string name="override_changes">Écraser les changements distants</string>
```

Comment:
Translations are handled at https://hosted.weblate.org/projects/syncthing/android, so I would suggest to remove the change from here and fix the text there.